Enhance dev setup

vgeorge opened the pull request:
Some non-urgent, proposed changes to the development setup:

- Added "yarn debug" command to open React Native Debugger more easily
- Added Redux DevTools composer to allow track application state in React Native Debugger
- Updated Cocoapods lockfiles, which was changed after running 'pod install'
- Added prettier configuration

I believe the project can benefit from using prettier as it helps with code consistency, improving  tracking and readability. I tried to make minimal changes to existing ESLint rules, the only one added was allowing dangling commas, which I think is [a good practice](https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8). No additional package was added, the developer can use its editor extension or the command line to format the code.

geohacker commented:
@vgeorge this PR looks good to me but a couple of things:
* can you remove changes to ios/Pods/Manifest.lock and ios/Podfile.lock. Not sure why we are on different version of cocoapods but it's better to ignore these for now.
* tests are failing, not sure how difficult those are. But if they are time consuming, I'd say we can put this on hold.

vgeorge commented:
@geohacker changes applied:

- Reversed lockfile
- Removed comma dangling rule
- Added VS Code launch config
- Replaced Redux DevTools with redux-logger
